codespell_lib/data/dictionary.txt

```diff
@@ -28237,9 +28246,15 @@ tranmitting->transmitting
 tranparent->transparent
 tranparently->transparently
 tranport->transport
+tranports->transports
```

Comment:
Would you mind adding entries tor:
`tranported->transported`  
`tranporting->transporting`